### PR TITLE
Add BananaPi BPI-Bit-S2 .

### PR DESCRIPTION
**Pull request by Wind-stormger**
Adds support for the BananaPi BPI-Bit-S2 Boards.

bpi_bit_s2

Based on esp32s2 chip.

with 25 ws2812 LEDs, 1 buzzer, 2 photosensitive sensors, 1 thermosensitive sensor. 

2 buttons on the front, 1 reset button and 1 boot button on the back.

### CODE REVIEW

File: ports/espressif/boards/bpi_bit_s2/board.c [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+bool board_requests_safe_mode(void) {
+    return false;
+}
+
+void reset_board(void) {
+
+}
+
+void board_deinit(void) {
+}
```
Comment (dhalbert) [inline ports/espressif/boards/bpi_bit_s2/board.c:48]: You can omit these definitions as of #6678.

```suggestion
// Use the MP_WEAK supervisor/shared/board.c versions of routines not defined here.
```

**Review by dhalbert — CHANGES_REQUESTED** [review]
Just a minor change, due to a recent PR.

File: ports/espressif/boards/bpi_bit_s2/board.c (hunk 1 above)
Comment (Wind-stormger) [reply ports/espressif/boards/bpi_bit_s2/board.c:48]: Oh, you mean #6878 ?
I was a little confused when I checked #6678, but it's ok, I got your point 😄 👌 .

Comment (dhalbert) [reply ports/espressif/boards/bpi_bit_s2/board.c]: Oops, yes, sorry, I did mean #6878.

**Comment by Wind-stormger** [comment]
@dhalbert I made force-pushed , combine two commits into one , no more changes.

**Review by dhalbert — APPROVED** [review]

**Comment by dhalbert** [comment]
Can you make a PR to https://github.com/adafruit/circuitpython-org now? Details in the README. Thanks.

**Comment by Wind-stormger** [comment]
Thanks for the reminder, I'm working on it.

**Pull request merged**